io/disk/fiotest.py

```diff
@@ -44,11 +45,25 @@ def setUp(self):
         """
         Build 'fio'.
         """
-        tarball = self.fetch_asset('http://brick.kernel.dk/snaps/fio-2.1.10.tar.gz')
+        url = "http://brick.kernel.dk/snaps/fio-2.1.10.tar.gz"
+        tarball = self.fetch_asset(url)
         archive.extract(tarball, self.srcdir)
         fio_version = os.path.basename(tarball.split('.tar.')[0])
         self.srcdir = os.path.join(self.srcdir, fio_version)
         build.make(self.srcdir)
+        self.disk = self.params.get('disk', default=None)
```

Comment:
I am working on it, just need to add a if statement which will take care if no disk is given in yaml file. And then you should be able to run it without passing any disk.